internal/handler/site/context/context.go

```diff
@@ -11,7 +11,7 @@ type contextValue struct {
 	Error error
 }
 
-func withSiteContext(ctx context.Context) context.Context {
+func WithSiteContext(ctx context.Context) context.Context {
```

Comment:
Why need move this file?

Reply:
because canon.go requires context, and putting context in site would lead to an import loop

Reply:
I mean originally there is no import loop, do you know which change that introduced this loop?